### ISSUE TITLE
Click [+] or [-] on shows

### ISSUE THREAD

**Issue by Zodiac-69** [namiman/horriblesubs_release_time_until#33]
I do not see the [+] or [-] in Firefox 62.0.2 (64-bit)
Am i doing something wrong???
![horriblesubs_release_schedule](https://user-images.githubusercontent.com/2076401/46029836-c685a280-c0f4-11e8-808b-d8b3f815c878.png)

**Comment by namiman** [comment]
Yep, that page seems to have changed. I might have some time to fix it someday...
